e2e/tests/home.spec.js

```diff
@@ -18,48 +22,18 @@ test.describe('home', () => {
     await password.press('Enter');
     await expect(page.getByText('Invalid email and/or password.')).toBeVisible();
   });
-  /*
-  test('redirects to EMS interface after EMS user login', async ({ page }) => {
-    await page.goto('/');
-    await page.getByLabel('Email').fill(process.env.EMS_USER);
-    const password = page.getByLabel('Password');
-    await password.fill(process.env.EMS_PASS);
-    await password.press('Enter');
-    await expect(page).toHaveURL('/ems');
-  });
-
-  test('redirects to ER interface after Hospital user login', async ({ page }) => {
-    await page.goto('/');
-    await page.getByLabel('Email').fill(process.env.HOSPITAL_USER);
-    const password = page.getByLabel('Password');
-    await password.fill(process.env.HOSPITAL_PASS);
-    await password.press('Enter');
-    await expect(page).toHaveURL('/er');
-  }); */
 
-  test('shows the Routed logo and two factor authentication form', async ({ page }) => {
-    await page.goto('/');
-    await page.getByLabel('Email').fill(process.env.HOSPITAL_USER);
-    const password = page.getByLabel('Password');
-    await password.fill(process.env.HOSPITAL_PASS);
-    await password.press('Enter');
-    await expect(page).toHaveURL('/auth/local/twoFactor');
-    await expect(page).toHaveTitle(/Routed/);
-    await expect(page.getByAltText('Routed logo')).toBeVisible();
-    await expect(page.getByText('Please enter the Authoritzation Code that was sent to your E-mail')).toBeVisible();
-    await expect(page.getByLabel('Code')).toBeVisible();
-    await expect(page.getByText('Submit')).toBeVisible();
+  test('Try Opening Mail Catcher', async ({ page }) => {
+    await page.goto(`http://${SMTP_HOST}:1080/`);
+    await expect(page.getByText('MailCatcher')).toBeVisible();
   });
 
-  test('Shows Error after Incorrect Two Factor Authentication', async ({ page }) => {
-    await page.goto('/');
-    await page.getByLabel('Email').fill(process.env.HOSPITAL_USER);
-    const password = page.getByLabel('Password');
-    await password.fill(process.env.HOSPITAL_PASS);
-    await password.press('Enter');
-    await expect(page).toHaveURL('/auth/local/twoFactor');
-    await page.getByLabel('Code').fill('123456');
-    await page.getByText('Submit').click();
-    await expect(page.getByText('Invalid Authoritzation Code.')).toBeVisible();
+  test('Try fetching messages from MailCatcher', async ({ request }) => {
+    const allMessagesRequest = await request.get(`http://${SMTP_HOST}:1080/messages`);
+    expect(allMessagesRequest.ok()).toBeTruthy();
+    const allMessages = await allMessagesRequest.json();
+    console.log(allMessages);
+    const deleteRequest = await request.delete(`http://${SMTP_HOST}:1080/messages`);
+    expect(deleteRequest.ok()).toBeTruthy();
```

Comment:
This seems to have passed, so I will be converting the `fetch` calls to implementations of `request`